tutorials/model-server-guide.md

```diff
@@ -0,0 +1,90 @@
+- [code/access.md](#access--usage-information---codeaccessmd)
+- [usage.md](#usage-disclaimer---usagemd)
+
+### Table of Contents - index.md
+
+Red Hat Developer Hub provides a Table of Contents on the left-hand side of the Tech Docs page, however, when you first open Tech Docs it may be helpful to display that information right in the face of the user.
+
+### Supported Models - models.md
+
+This document is intended to consolidate all of the supported models that are accessible via this model server. This allows users to easily view the model information (such as Hugging Face link) and the documentation (tech docs link) for each model they have the option to use. By linking to the deployed models resource Tech Docs we are able to reuse the documentation and utilize the full power of the Tech Docs.
+
+### Access & Usage Information - code/access.md
+
+This document should aim to provide users with the information required to consume the models through the model server. This would contain basic setup information such as how to access the server as well as getting yourself authenticated. Additionally, it would be beneficial to contain examples of how you can utilize the model server endpoint to query the models. Remember to keep API keys out of your documentation if possible.
+
+This document would ideally contain a separate section for each supported model to ensure that any differences in usage across models is accounted for.
+
+You are able to format Markdown to properly highlight code based on language by adding the language next to your 3 back-ticks.
+
+Example: 
+````
+```python
+<code block>
+```
+````
+
+### Usage Disclaimer - usage.md
+
+This section is important for the users of the model server (and models) to have the information necessary to make smart decisions while consuming these services. This is a good section for outlining any legal disclaimers or notices you wish to attach to the services. 
+
+As an example, if you do not want users generating open source or production software with the models you should add a snippet about that to this document.
+
+
+## Creating Tech Docs
+
+- [Directory Format & Best Practices](#directory-format--best-practices)
+- [Creating mkdocs.yaml](#creating-mkdocsyaml)
+- [Adding Docs To The Catalog](#adding-docs-to-the-catalog)
+
+### Directory Format & Best Practices
+
+When looking to add Tech Docs to your resource there are a few practices you should adhere to, this will help ensure that everything is rendered correctly:
+
+- All documentation should be under `/docs` with image assets such as PNGs being stored under `/docs/images` and Markdown files containing code samples under `/docs/code`
+
+![Example Structure Img](../assets/techdoc-structure-example.png)
+
+### Creating mkdocs.yaml
+
+- All Markdown files should be referenced within mkdocs.yaml, for file path you should reference the .md files as if mkdocs.yaml is the root:
+
+![Example Structure Img](../assets/mkdocs-structure-example.png)
```

Comment:
I think the screenshot could be wider here:
![image](https://github.com/user-attachments/assets/c02262ba-9e4c-4212-904b-d76df955bd9c)

Reply:
We could also use yaml code?